config/puma.rb

```diff
@@ -12,15 +12,15 @@
 # Starting with a low number of workers and threads provides adequate
 # performance for most applications, even under load, while maintaining a low
 # risk of overusing memory.
-workers Integer(ENV.fetch('WEB_CONCURRENCY', 2))
-threads_count = Integer(ENV.fetch('MAX_THREADS', 2))
+workers Integer(Figaro.env.web_concurrency || 2)
```

Comment:
Are we even using Puma in production? I thought all our Rails apps were configured to use Passenger? https://github.com/18F/identity-dashboard/blob/7ec7dad09650c9b6c08ce81c7e216ffa5187acaf/config/deploy.rb#L16-L18